### PR TITLE
Go / Js lib updates

### PR DESCRIPTION
**Pull request by loleg**
This is just couple of routine updates to the backend and frontend dependencies.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM debian:stable-slim
```
Comment (loleg) [inline Dockerfile:1]: We had an issues with the stable-slim build.

File: .circleci/config.yml [hunk 2]
```diff
@@ -342,16 +291,3 @@ workflows:
           filters:
             branches:
               only: main
-      - approve-handelsregister:
```
Comment (loleg) [inline .circleci/config.yml:345]: No longer in use.

File: .circleci/config.yml [hunk 3]
```diff
@@ -219,60 +219,9 @@ jobs:
           command: |
             echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
             docker push $CONTAINER_IMAGE_NAME:latest
-
```
Comment (loleg) [inline .circleci/config.yml:222]: Staging deployment via CircleCI is legacy.

**Comment by tafonina** [comment]
@loleg checks are not passed. validate-go is failed

**Pull request closed without merging**